USE_CASES.md

```diff
@@ -1392,19 +1394,19 @@ To create a Domain Whitelabel Via the API:
 package main
 
 import (
-	"fmt"
-	"log"
-	"os"
+  "fmt"
```

Comment:
Please revert these indentation examples, the current version is from `go fmt` which is our standard for formatting.